compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/window/ComposeWindow.web.kt

```diff
@@ -304,6 +312,13 @@ internal class ComposeWindow(
             processKeyboardEvent(event)
         }
 
+
+        state.globalEvents.addDisposableEvent("dragover") { event ->
```

Comment:
Why should it be outside DnD manager?

Reply:
@MatkovIvan Here we are describing how the parent container of canvas - in this particular case - window - manages situations which canvas can not handle per se - this basically instructs browser how it should (visually) process drag and drop operations that we cancelled. 

But!  While I was writing this down I've realised that there's a room for improvement and will comment on modified code as soon as it will be pushed